### ISSUE TITLE
No RCS can be assigned to detected stationary objects

### ISSUE THREAD

**Issue by hschoen** [OpenSimulationInterface/open-simulation-interface#462]
Whereas an RCS can be assigned to detected moving objects this is not possible for stationary objects (and traffic lights/signs).

For detected moving objects the related messages are:

`optional RadarSpecificObjectData | radar_specifics = 100`

`optional double osi3::RadarSpecificObjectData::rcs = 1`

**Comment by pmai** [comment]
I agree that the sensor specific fields (not only for radar) should also be added to DetectedStationaryObject. This would be an issue for the SensorModeling group, and could be done backward compatibly for a 3.4, for example.

**Comment by kmeids** [comment]
@hschoen, I agree as well!
Is it possible for you to join one of our Sensor Modeling WG meetings?
They take place on a Bi-weakly basis on Friday from 9:00 to 10:00 CET.

**Comment by hschoen** [comment]
Hello,

yes, I can join one of your Sensor Modeling WG meetings. Could you forward me a meeting invitation? I could join this week, but not on 5 Feb.

Mit freundlichen Grüßen / Best regards

Dr. Helmut Schoen

HF-Sensoren, EMV (CR/AEL1)
Robert Bosch GmbH | Postfach 10 60 50 | 70049 Stuttgart | GERMANY | www.bosch.com
Tel. +49 711 811-10880 | Mobil +49 173 9187997 | Telefax +49 711 811 | Threema<threema://add/?id=K286UVMU> / Threema Work<threemawork://add/?id=K286UVMU>: K286UVMU | Helmut.Schoen@de.bosch.com<mailto:Helmut.Schoen@de.bosch.com>

Sitz: Stuttgart, Registergericht: Amtsgericht Stuttgart, HRB 14000;
Aufsichtsratsvorsitzender: Franz Fehrenbach; Geschäftsführung: Dr. Volkmar Denner,
Prof. Dr. Stefan Asenkerschbaumer, Filiz Albrecht, Dr. Michael Bolle, Dr. Christian Fischer,
Dr. Stefan Hartung, Dr. Markus Heyn, Harald Kröger, Rolf Najork, Uwe Raschke
​
Von: Kmeid <notifications@github.com>
Gesendet: Mittwoch, 27. Januar 2021 13:28
An: OpenSimulationInterface/open-simulation-interface <open-simulation-interface@noreply.github.com>
Cc: Schoen Helmut (CR/AEL1) <Helmut.Schoen@de.bosch.com>; Mention <mention@noreply.github.com>
Betreff: Re: [OpenSimulationInterface/open-simulation-interface] No RCS can be assigned to detected stationary objects (#462)


@hschoen<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fhschoen&data=04%7C01%7Chelmut.schoen%40de.bosch.com%7Cda9076e45c6c40ac0c6008d8c2bf04d3%7C0ae51e1907c84e4bbb6d648ee58410f4%7C0%7C0%7C637473472958112149%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=rNpkTNRy2uDDgkq7Na3M89W5a0c2tPmV2T2GZVKM%2BCs%3D&reserved=0>, I agree as well!
Is it possible for you to join one of our Sensor Modeling WG meetings?
They take place on a Bi-weakly basis on Friday from 9:00 to 10:00 CET.

**Comment by kmeids** [comment]
I'll set up a one on one meeting for this week and update the rest of the group next week (it is a bi-weekly meeting and initially we don't have it programmed for this week).

**Comment by hschoen** [comment]
OK. Alternatively I can join on 19 Feb.

Mit freundlichen Grüßen / Best regards

Helmut Schoen
CR/AEL1

Tel. +49 711 811-10880 | Mobil +49 173 9187997 | Threema Work<threemawork://add/?id=K286UVMU>: K286UVMU
​
Von: Kmeid <notifications@github.com>
Gesendet: Mittwoch, 27. Januar 2021 14:16
An: OpenSimulationInterface/open-simulation-interface <open-simulation-interface@noreply.github.com>
Cc: Schoen Helmut (CR/AEL1) <Helmut.Schoen@de.bosch.com>; Mention <mention@noreply.github.com>
Betreff: Re: [OpenSimulationInterface/open-simulation-interface] No RCS can be assigned to detected stationary objects (#462)


I'll set up a one on one meeting for this week and update the rest of the group next week (it is a bi-weekly meeting and initially we don't have it programmed for this week).

**Comment by stefancyliax** [comment]
OSI CCB:
@kmeids What is the state here? Can this be finished for 3.4?

**Comment by kmeids** [comment]
@stefancyliax, so far we did not get any further activities on this topic and we were still waiting on updates from other project.
I'll move it to 4.0.

**Comment by thomassedlmayer** [comment]
Will be resolved by #708. For StationaryObjects we already have sensor specifics.
